src/layout/Grid/GridComponent.tsx

```diff
@@ -131,6 +156,8 @@ function InternalRow({ header, readOnly, children }: InternalRowProps) {
 interface CellProps {
   className?: string;
   columnStyleOptions?: ITableColumnProperties;
+  referenceComponent?: LayoutNode;
+  help?: string;
 }
```

Comment:
Could probably just make another type for `<CellWithLabel>`. Mixing these two cell types could just be confusing.